src/unassigned-cases/converters/case-converter.ts

```diff
@@ -0,0 +1,77 @@
+import { SearchResultViewItem } from '@hmcts/ccd-case-ui-toolkit';
```

Comment:
This entire file looks like a helper for the presentation layer. Why not perhaps convert to pipes and call these directly in template?